src/helpers/helpers.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const getNewId = (data) => {
+  return Math.max(...data.map((item) => item.id), 0) + 1;
```

Comment:
Change the starting value from 0 to - in Math.max() to avoid returning 0 as the first new id.